src/main/rx/rx.c

```diff
@@ -539,6 +541,52 @@ static void detectAndApplySignalLossBehaviour(void)
 #endif
 }
 
+#define MAX_RC_HISTORY_ITEMS 4
+
+uint16_t rcHistory[MAX_SUPPORTED_RC_CHANNEL_COUNT][MAX_RC_HISTORY_ITEMS];
+
+void updateChannelStability(void)
+{
+    for (uint8_t channelIndex = 0; channelIndex < MAX_SUPPORTED_RC_CHANNEL_COUNT; channelIndex++) {
+
+        // calculate the maximum absolute delta between the new reading each each of the previous readings.
+        uint16_t maximumDelta = 0;
+        for (int8_t historyIndex = -1; historyIndex < MAX_RC_HISTORY_ITEMS - 1; historyIndex++) {
```

Comment:
yeah, there's a few approaches we could use for stability.  I initially picked maximum delta as it seemed to fit the bill, I didn't consider max-min or max delta - min delta.

![stability-calculations-1](https://cloud.githubusercontent.com/assets/57075/12510332/64a402a2-c109-11e5-9bc4-9b904c64f3b0.JPG)